packages/python/plotly/plotly/express/_imshow.py

```diff
@@ -60,6 +61,9 @@ def imshow(
     labels={},
     x=None,
     y=None,
+    animation_frame=None,
+    facet_col=None,
+    facet_col_wrap=None,
```

Comment:
would be good to add `facet_col_spacing` and `facet_row_spacing`

Reply:
done